ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Source.js

```diff
@@ -311,7 +326,7 @@ define([
             parseServiceModel: function () {
                 var resModel = this;
                 var collection = resModel.get('collection');
-                collection.reset();
+                collection.removeAllSources();
```

Comment:
What's the motivation to switch here?

Reply:
`collection.reset()` doesn't trigger remove events on the source models, and I changed the source model to clean up polling events when removed.